analyzers/its/expected/NetCore31/NetCore31--S1128.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
The file name `NetCore31--S1128.json` seems to be missing the project GUID. Does the script works well? I remember that the scanner complains if the project does not have a GUID.